Config/build failure on macOS 12.6.2

I'm trying to do a build on macOS 12.6.2

Starting from a clean clone of the GIT repo:

cd cvc5
./configure.sh --static --auto-download --cryptominisat

finishes with

```
-- Configuring done
CMake Error: The following variables are used in this project, but they are set to NOTFOUND.
Please set them or make sure they are set and tested correctly in the CMake files:
/Users/rodchap/Desktop/rod/tools/src/cvc5/src/parser/FLEX_INCLUDE_DIR
   used as include directory in directory /Users/rodchap/Desktop/rod/tools/src/cvc5/src/parser

-- Generating done
CMake Generate step failed.  Build files cannot be regenerated correctly.
```

If I look in src/parser/CMakeLists.txt, I find

```
if(CMAKE_SYSTEM_NAME STREQUAL "Darwin")
  list(APPEND CMAKE_PREFIX_PATH "/usr/local/opt/flex")
endif()
```

but my system has Homebrew installed below /opt/homebrew, not /usr/local

If I change that line to 

```
  list(APPEND CMAKE_PREFIX_PATH "/opt/homebrew/opt/flex")
```

then it seems to work.

--- a/src/parser/CMakeLists.txt
+++ b/src/parser/CMakeLists.txt
@@ -20,7 +20,8 @@ find_package(Java COMPONENTS Runtime REQUIRED)
 
 # Search for flex installed via homebrew.
 if(CMAKE_SYSTEM_NAME STREQUAL "Darwin")
-  list(APPEND CMAKE_PREFIX_PATH "/usr/local/opt/flex")
+  list(APPEND CMAKE_PREFIX_PATH "/usr/local/opt/flex") # macOS Intel
+  list(APPEND CMAKE_PREFIX_PATH "/opt/homebrew/opt/flex") # macOS Apple Silicon
 endif()
 find_package(FLEX REQUIRED)
 

